example/device/framegrabber/readRealSenseData.cpp

```diff
@@ -57,8 +58,19 @@
 #include <visp3/io/vpVideoWriter.h>
 
 #if defined(VISP_HAVE_PCL)
+#include <pcl/pcl_config.h>
+#if defined(VISP_HAVE_PCL_COMMON)
+#if PCL_VERSION_COMPARE(>=,1,14,1)
+#include <pcl/impl/point_types.hpp>
+#else
+#include <pcl/point_types.h>
+#endif
```

Comment:
I'm confused with the following code
```
#if PCL_VERSION_COMPARE(>=,1,14,1)
#include <pcl/impl/point_types.hpp>
#else
#include <pcl/point_types.h>
#endif
```

Is this really needed since in some other files we have only
```
#include <pcl/point_types.h>
```
and it seems to build.

Reply:
I hadn't noticed that. Modified in 3329f0310